allo/customize.py

```diff
@@ -1251,6 +1251,7 @@ def customize(
     global_vars: dict = None,
     instantiate: list = None,
     context: Context = None,
+    typing_rule_set="default",
     unroll: bool = True,
 ) -> Schedule:
```

Comment:
Could you also help add a docstring to document what the arguments mean in this `customize` function?